packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -290,6 +290,21 @@ export class IosSimulatorDevice extends DeviceBase {
     }
   }
 
+  terminateApp = async (bundleID: string) => {
```

Comment:
NIT:
I personally dislike arrow function class members, I find them hard to parse as a method at a glance when reading the source.
I'd prefer if this was a normal method and later in the `map` you wrap it in an arrow function or `.bind(this)` it.